playground.sh

```diff
@@ -177,27 +258,22 @@ docker)
   runtime="docker";
   ;;
 *)
-  echo "ERROR: please specify which runtime you want to use, available runtime: [docker|k8s]" 
+  echo "[ERROR] please specify which runtime you want to use, available runtime: [docker|k8s]"
+esac
+
+skipChecks=false
+
+case "$3" in
+--skip-checks)
+  skipChecks=true;
+  ;;
+-s)
+  skipChecks=true;
+  ;;
```

Comment:
Do we need `skipChecks` startup param? It feels too complicated.
I think we must check docker or k8s envorinments.

Reply:
Thank you for the feedback. I'll remove the skipChecks parameter as suggested. The test didn't take much time, so this change should be straightforward.